CleverPush/Source/CleverPushInstance.m

```diff
@@ -3678,7 +3678,7 @@ - (void)showAppBanner:(NSString*)bannerId notificationId:(NSString*)notification
 - (void)showAppBanner:(NSString*)bannerId channelId:(NSString*)channelId notificationId:(NSString*)notificationId {
```

Comment:
Please change https://github.com/cleverpush/cleverpush-ios-sdk/blob/f0180f3eb1f6ac562d5a30ba06ddcf878faba118/CleverPush/Source/CleverPushInstance.m#L3675 to `YES` instead of `true`